docs/other/Caching.md

```diff
@@ -1,33 +0,0 @@
-# Caching
```

Comment:
Why are we removing the guide here?

Reply:
We made the decision to remove all markdown docs in favour of Go docs.